http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -305,32 +296,42 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       gkeClusterId = dbCluster.getGkeClusterId
       googleProject = params.googleProject
 
-      _ <- logger.info(ctx.loggingCtx)(
-        s"Beginning app creation for app ${app.appName.value} in cluster ${gkeClusterId.toString}"
-      )
-
       // Create namespace and secrets
       _ <- logger.info(ctx.loggingCtx)(
         s"Creating namespace ${namespaceName.value} and secrets for app ${app.appName.value} in cluster ${gkeClusterId.toString}"
       )
 
       _ <- kubeService.createNamespace(gkeClusterId, KubernetesNamespace(namespaceName))
-      secrets <- getSecrets(namespaceName)
-      _ <- secrets.parTraverse(secret =>
-        kubeService.createSecret(gkeClusterId, KubernetesNamespace(namespaceName), secret)
-      )
 
       // Create KSA
       ksaName = config.galaxyAppConfig.serviceAccount
       gsa = dbApp.app.googleServiceAccount
-      annotations = Map("iam.gke.io/gcp-service-account" -> gsa.value)
-      ksa = KubernetesModels.KubernetesServiceAccount(ksaName, annotations)
 
-      _ <- logger.info(ctx.loggingCtx)(
-        s"Creating Kubernetes service account ${ksaName.value} for app  ${app.appName.value} in cluster ${gkeClusterId.toString}"
+      // Resolve the cluster in Google
+      googleClusterOpt <- gkeService.getCluster(gkeClusterId)
+      googleCluster <- F.fromOption(
+        googleClusterOpt,
+        ClusterCreationException(
+          s"Cluster not found in Google: ${gkeClusterId} | trace id: ${ctx.traceId}"
+        )
       )
-      _ <- kubeService.createServiceAccount(gkeClusterId, ksa, KubernetesNamespace(namespaceName))
 
+      nfsDisk <- F.fromOption(
+        dbApp.app.appResources.disk,
+        AppCreationException(s"NFS disk not found in DB for app ${app.appName.value} | trace id: ${ctx.traceId}")
+      )
+
```

Comment:
Maybe add a log "installing set-up chart"

(There are a lot of steps here and I find logging each step useful)

Reply:
there will be the following log already...do you still want the additional log? I found it a bit too noisy when each step prints out something
```
2021/02/18 17:29:55 creating 3 resource(s)
2021/02/18 17:29:55

Finished installing release:  ya8b8r-gxy-rls-setup-rls
{"@timestamp":"2021-02-18T17:29:55.075Z","@version":"1","message":"The command 'helm installChart' succeeded","logger_name":"org.broadinstitute.dsde.workbench.leonardo.http.Boot","thread_name":"ioapp-compute-4","severity":"INFO","level_value":20000}
```

Reply:
I think I'd still like an additional `log.info`… the helm client logs tend to be in a different format I think and may be hard to parse (e.g. with `TraceId`).